templates/taxi/car_list.html

```diff
@@ -6,12 +6,19 @@
     <ul>
       {% for car in car_list %}
         <li>
-          <a href="{% url "taxi:car-detail" pk=car.id %} ">{{ car.id }}</a>
+          <a href="{% url "taxi:car-detail" pk=car.id %}">{{ car.id }}</a>
           {{ car.model }} ({{ car.manufacturer.name }})
+          <a href="{% url 'taxi:car-update' pk=car.id %}">
+            <i class="bi bi-pencil"></i>
+          </a>
+          <a href="{% url "taxi:car-delete" pk=car.id %}">
```

Comment:
Inconsistent use of quotes in the `{% url %}` tag. It's better to use single quotes consistently, as in `{% url 'taxi:car-delete' pk=car.id %}`.